### PR TITLE
setup icons to all tabs.

### PR DESCRIPTION
**Pull request by YaroslavP1995**
Setup those icons to all tabs. Please review my code.

### CODE REVIEW

File: app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockActivity.java [hunk 1]
```diff
@@ -18,27 +18,31 @@
...
-    public void setPagerAdapter(List<Fragment> fragmentList, List<String> titleList) {
-        if (fragmentList != null && titleList != null && fragmentList.size() == titleList.size()) {
+    public void setPagerAdapter(List<Fragment> fragmentList, List<String> titleList, List<Integer> iconList) {
+        if (fragmentList != null && titleList != null && iconList != null && fragmentList.size() == titleList.size()) {
             TabAdapter adapter = new TabAdapter(getSupportFragmentManager());
             for (int i = 0; i < fragmentList.size(); i++) {
                 adapter.addFragment(fragmentList.get(i), titleList.get(i));
             }
             viewPager.setAdapter(adapter);
+            for (int j = 0; j < iconList.size(); j++){
```
Comment (Artem-Zelinka) [inline app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockActivity.java]: move if with check that iconList != null here and compare it's size with size of tabs in layout

File: app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockPresenter.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
     private SimpleClockView simpleClockView;
     private Context context;
 
-   SimpleClockPresenter(Context context) {
+    SimpleClockPresenter(Context context) {
```
Comment (Artem-Zelinka) [inline app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockPresenter.java:23]: Remove this extra space

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -12,11 +12,13 @@
...
-            app:tabIndicatorColor="@color/colorPrimaryDark"
-            app:tabIndicatorHeight="4dp"
-            app:tabBackground="@color/colorPrimary"
             app:tabMode="fixed"
-    >
+            app:tabTextColor="@color/whiteViewColor"
+            app:tabSelectedTextColor="@color/primaryViewColor"
+            app:tabIndicatorColor="@color/primaryViewColor"
+            app:tabIconTint="@color/primaryViewColor"
+            app:tabIndicatorHeight="4dp"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout/activity_main.xml]: remove height or move 4dp to dimens

**Review by Artem-Zelinka — CHANGES_REQUESTED** [review]
Small changes

**Comment by YaroslavP1995** [comment]
Fix syntax errors, added size check .

**Review by Artem-Zelinka — APPROVED** [review]

**Pull request merged**